tests/src/test/scala/whisk/core/cli/test/ApiGwRestBasicTests.scala

```diff
@@ -36,7 +36,7 @@ import common.WskProps
 @RunWith(classOf[JUnitRunner])
 abstract class ApiGwRestBasicTests extends BaseApiGwTests {
 
-  val clinamespace = wsk.namespace.whois()
+  lazy val clinamespace = wsk.namespace.whois()
```

Comment:
is it best practice to use `lazy` for abstract class property initialization?

Reply:
required to avoid NPE when there are overrides.